src/sentry/integrations/api/endpoints/external_team_details.py

```diff
@@ -45,19 +50,24 @@ def convert_args(
         )
         return args, kwargs
 
+    @extend_schema(
+        operation_id="Update an External Team",
+        parameters=[
+            GlobalParams.ORG_ID_OR_SLUG,
+            GlobalParams.TEAM_ID_OR_SLUG,
+            OrganizationParams.EXTERNAL_TEAM_ID,
+        ],
+        request=ExternalTeamSerializer,
+        responses={
```

Comment:
is it right that the request and response serializer are the same? seems like we're not using the sentry serializer for the response?

Reply:
guess we're using the DRF one? a rairity at sentry haha

Reply:
we end up returning the same result as what the serializer returns, so decided to just use that instead of defining another dataclass.